operations/node_management/Reboot_NCNs.md

```diff
@@ -174,6 +174,16 @@ Execute the rolling NCN reboot procedure steps for the particular node type bein
        echo "${xname}"
        cray cfs v3 components update "${xname}" --enabled False --error-count 0
    done
+    ```
+
+   Alternatively, disable multiple components at once by using the `updatemany` command.
+   Refer to the [Managing many components](../configuration_management/CFS_Commands_Cheat_Sheet.md#managing-many-components)
+   for `updatemany` command requirements.
```

Comment:
````suggestion
   ```
   Alternatively, if running CSM 1.6.2 or higher, disable multiple components at once by using the `updatemany` command. Refer to
   [Managing many components](../configuration_management/CFS_Commands_Cheat_Sheet.md#managing-many-components)
   for complete `updatemany` command requirements and usage.
````